azkaban-web-server/src/test/java/azkaban/restli/DeployProjectTest.java

```diff
@@ -49,6 +49,7 @@ public void testWarnedDeploy() {
   }
 
   @Test
+  @SuppressWarnings("MissingFail")
```

Comment:
Would adding an Assert.fail(); statement at the end of the try block as the examples in 
http://errorprone.info/bugpattern/MissingFail work better?

```java
 public void expectedException_withFail() {
    try {
      dummyMethod();
      Assert.fail();
    } catch (Exception expected) {
    }
  }
```